src/app/courses/courses.service.ts

```diff
@@ -24,6 +24,7 @@ export class CoursesService {
   stepIndex: any;
   returnUrl: string;
   currentParams: any;
+  myCourseIndexes: any= [];
```

Comment:
You should have spaces around the `=`.  I'm not sure why this isn't being picked up by the linter, but CodeClimate is detecting it at least.